### ISSUE TITLE
Occasional "forget entry" messages on unmount

### ISSUE THREAD

**Issue by sector-f** [wfraser/fuse-mt#15]
My FUSE program occasionally prints messages to the terminal upon being unmounted. Here's an example:

```
forget entry InodeTableEntry { path: Some("/LambdaComplex/out"), lookups: 2, generation: 0 }
```

Based on a quick search of this repo for "forget entry", I assume that [this](https://github.com/wfraser/fuse-mt/blob/97ba28f07928837ff5e53bcd6dc561b5e3241013/src/inode_table.rs#L167) is the code that's causing it. Perhaps that `println!` should be a `debug!`?

**Comment by sector-f** [comment]
Hopefully fixed in #16

**Comment by wfraser** [comment]
Thanks for reporting! Fixed in 92131b0e3274e39959cf118aad63019bd78fbee8

**Comment by albel727** [comment]
Maybe release a new version of fuse-mt with this fix to crates.io?

**Comment by wfraser** [comment]
Done. Published as v0.4.2

**Comment by albel727** [comment]
Thanks!
